app/src/format.py

```diff
@@ -369,13 +380,24 @@ def reify_citations(
 
 # FIXME: Refactor to reduce code replication with replace_citation_ids()
 def _add_citation_links(
-    response: str, remapped_citations: dict[str, Subsection], config: FormattingConfig
+    response: str,
+    remapped_citations: dict[str, Subsection],
+    config: FormattingConfig,
+    list_of_accordion_ids: list[dict] | None,
```

Comment:
I don't think you need to accept `None` here, you could require an empty list (or dictionary)